build: adds intercom update with arb and mainnet usd balance values

matthew-garrett opened the pull request:
- Adds intercom update call passing wallet address, arbitrum and mainnet use wallet balance values

jeffywu commented on apps/web/src/containers/App/App.tsx:
```diff
@@ -269,6 +269,7 @@ export const App = () => {
+
   const notionalTheme = useNotionalTheme(themeVariant);
 
   return (
```
You don't need the Intercom Provider in here anymore?

matthew-garrett replied:
Good call, it must have gotten washed out when I rebased

jeffywu commented on packages/shared/wallet/src/hooks/use-intercom-update.tsx:
```diff
@@ -0,0 +1,38 @@
+      update({
+        userId: selectedAccount,
+        name: selectedAccount,
+        customAttributes: {
```
You don't need to destructure here? Can't you just do `customAttributes: networkBalanceData`

matthew-garrett committed:
build: adds intercom update with arb and mainnet usd balance values

build: move intercom update logic to a hook

build: removes log

build: fixes app file to use intercom

build: removes log

build: clean up
